measures/301EnergyRatingIndexRuleset/tests/test_hvac.rb

```diff
@@ -474,30 +474,6 @@ def test_programmable_thermostat
     _check_thermostat(hpxml_doc, "programmable thermostat")
   end
 
-  def test_dse
-    hpxml_name = "base-hvac-dse.xml"
```

Comment:
This file is missed in sample_files, so I removed this test. But maybe it's still necessary for ERI to check if these four configurations could generate DSE inputs correctly. Any suggestion where to rake it (or is there already a dse file to check)?

Reply:
I guess now that we had to revert the DSE inputs in the EPvalidator.rb, we should also revert the removal of this base-hvac-dse.xml file. Can you make that change? Then you can bring this test back.

Reply:
Sure